reana_commons/openapi_specifications/reana_job_controller.json

```diff
@@ -174,10 +171,7 @@
                   },
                   "2e4bbc1d-db5e-4ee0-9701-6e2b1ba55c20": {
                     "cmd": "date",
-                    "cvmfs_mounts": [
-                      "atlas.cern.ch",
-                      "atlas-condb.cern.ch"
-                    ],
+                    "cvmfs_mounts": "[\"atlas.cern.ch\", \"atlas-condb.cern.ch\"]",
```

Comment:
The same change in `./reana-job-controller/reana_job_controller/rest.py`?

Reply:
Yes, it is there https://github.com/reanahub/reana-job-controller/pull/111/files#diff-823cadb617b85239e4292c822735b84aR123